docs/content/direct/core-chart.md

```diff
@@ -181,6 +200,32 @@ After the initial installation is completed, there are two main ways to install
       --set-json='WDSes=[{name":"wds2"}]'
     ```
 
+The core chart also supports the use of external clusters as ITS Control Planes. A specific application is to connect to existing OCM clusters. As an example, create a first local kind cluster with OCM installed in it:
+
+```shell
+kind create cluster --name ext1
+
+clusteradm init
+```
+
+Then, create a second kind cluster suitable for KubeStellar installation and create a bootstrap secret in the new cluster with the kubeconfig information of the `ext1` cluster:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubeflex --port 9443
+
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/create-external-bootstrap-secret.sh) --controlplane its1 --context kind-ext1 --address https://ext1-control-plane:6443 --verbose

```

Comment:
That `--address https://ext1-control-plane:6443` part is black magic unless explained. I might add an explanation along the following lines.

The `--address` URL needs to be one that the KubeFlex controller can use to open a connection to the external cluster's Kubernetes apiserver(s). In this example, the external cluster is a `kind` cluster with one kube-apiserver and it listens on port 6443. We rely on the DNS resolver in Docker networking to map the domain name `ext1-control-plane` to the address of the container of that same name on the relevant Docker network.